File: src_20170503/src/web/web/fabs/services/choose_service.js
ysp.service('chooseService', chooseService);
chooseService.$inject = ['$http'];

function chooseService($http) {
    return {
            choose_depcpremove: function (data) {
                    return $http.post(base_url + '/chooseservice/choose_depcpremove', data);
             },
             choose_ebankpremove: function (data) {
                return $http.post(base_url + '/chooseservice/choose_ebankpremove', data);
             },
             choose_loanpremove: function (data) {
                return $http.post(base_url + '/chooseservice/choose_loanpremove', data);
             }

    }
}

File: src_20170503/src/web/server/fabs/views/chooseservice.py
# -*- coding: utf-8 -*-
"""
yinsho.api.seims
#####################

yinsho seims view module
"""

from flask import Blueprint, request, json, abort, current_app, jsonify, session as web_session, g

from ..services import ChooseService
from . import route


bp = Blueprint('chooseservice', __name__, url_prefix='/chooseservice')

chooseService = ChooseService()
'存款客户号移交'
@route(bp, '/choose_depcpremove',methods=['POST'])
def choose_depcpremove ():
    current_app.logger.debug(request.json)
    return chooseService.choose_depcpremove(**request.json)

'电子银行移交'
@route(bp, '/choose_ebankpremove',methods=['POST'])
def choose_ebankpremove ():
    current_app.logger.debug(request.json)
    return chooseService.choose_ebankpremove(**request.json)

'贷款移交'
@route(bp, '/choose_loanpremove',methods=['POST'])
def choose_loanpremove ():
    current_app.logger.debug(request.json)
    return chooseService.choose_loanpremove(**request.json)

